sites/tvpassport.com/tvpassport.com.channels.xml

```diff
@@ -599,7 +620,8 @@
     <channel lang="es" xmltv_id="KVEADT1.us" site_id="telemundo-kvea-los-angeles-ca/2611">Telemundo (KVEA) Los Angeles, CA</channel>
     <channel lang="es" xmltv_id="W20CQD2.us" site_id="esperanza-w20cqd2-hempstead-ny/16437">Esperanza (W20CQ-D2) Hempstead, NY</channel>
     <channel lang="es" xmltv_id="WASALD1.us" site_id="estrella-wasald-port-jervis-ny/11071">Estrella (WASA-LD) Port Jervis, NY</channel>
-    <channel lang="es" xmltv_id="WFTYDT2.us" site_id="uni-wftydt2-new-york-ny/1962">UNI (WFTY-DT2) New York, NY</channel>
+    <channel lang="es" xmltv_id="WFTYDT2.us" site_id="uni-wftydt2-new-york-ny/1962">UniMás (WFTY-DT2) New York, NY</channel>

```

Comment:
Database have this as Univision - https://iptv-org.github.io/?q=WFTYDT2.us

Reply:
https://en.wikipedia.org/wiki/WFTY-DT

Wikipedia has DT2 (67.2) as UniMás. (see other comment for relevant quote)

Reply:
https://github.com/iptv-org/database/pull/2164